parquet-hadoop/pom.xml

```diff
@@ -85,6 +85,11 @@
       <artifactId>commons-pool</artifactId>
       <version>1.6</version>
     </dependency>
+    <dependency>
+      <groupId>it.unimi.dsi</groupId>
```

Comment:
The `parquet-hadoop` actually depends on the `fastutil` package, without this package I was unable to test the project in isolation.